templates/includes/beta-banner.html

```diff
@@ -1,9 +1,11 @@
+{% load wagtailcore_tags %}
 <div class="beta-banner" aria-label="About The National Archives' beta">
```

Comment:
I'm not sure that `aria-label` is needed here because the `<div>` itself is not an interactive element.

Reply:
Thanks @gtvj, I can remove this. I did see this pattern used elsewhere in Etna so we might want to look at those too as well.